_posts/2018-03-10-meetup-march.md

```diff
@@ -0,0 +1,108 @@
+
+Another argument raised was that vibranium is of critical use in
+Wakanda. It is used everywhere, which kinda implies that the only
+way Africa can develop to that level is if it had some divine
+intervention.
+
+For a movie that was primarily of black cast, an argument was
+raised that it was pretty good. The success of the movie would
+lead to more inclusion in Hollywood. A counter to that was if that
+was what blacks could come up with, with all the resources given,
+it was pretty disappointing.
+
+# Animes discussed:
+
+As with all book clubs, anime was discussed. Key was One Piece,
+Full Metal Alchemist, ReZero and Death Note.
+
+One Piece was hailed as one of the greatest anime of all time.
+However, it currently clocks over 800 episodes (around 280 Hrs),
+which is a great obstacle to get over. One of the beauties of
+anime is the implications the story line has like the Gorosei
+wanting to have beautiful slaves, the racism with the fishmen,
+luffy's decision to search for freedom and many more. For
+potential fans, you're advised to start watching it because it's
+only going to get longer and longer.
+
+One Piece: freedom, slavery
+Full Metal Alchemist
+
+# Books discussed
+
+## Harry Potter and the methods of Rationality
+This takes a turn at the Harry Potter series. It is written by
+Eliezer Yudkowsky. It takes an approach where Harry Potter is
+actually smart and makes better decisions than in the original
+books. Those who read it claimed it changes their view of the
+original book.
+
+## Wheel of Time series
+This is a 13 book series, written between 1990 and 2013. Each
+blocks has around 900 pages. I'm currently reading book 2 and its
+a pretty interesting fantasy series. Also since the series is
+done, I won't have to wait another 6 years like the Game of
+Thrones for another book.
+
+## Other books
+
+Katherine Boo's Behind the Beautiful Forevers came up again. It is
+a beautiful story set in India that describes the concept of a
+glass ceiling. It is a beautiful story that is more set in
+hopeless reality than happy endings.
+
+# General
+For IT practitioners in Kenya, some people feel frustrated by what
+their work mate's lack of effort to try to fix common problems
+causes. This has led to most being condenscending to their work
+mates. This should however not be case. This is because if these
+other poeple took up this effort, it would end up in job loss for
+others. Also while it can be really frustrating to fix the same
+thing for the same person day in day out, we should try to be
+understanding because that is not the specialty of the person.
+
+What should happend when some one achieves all their goals in
+line? Should they just die? Should they kill themselves or
+continue living their lives (which will be pointless at this
+point)? Leo Tolstoy's book deal with existentialism, so one might
+find answers there.
+
+When someone is successful at something, the first praise is
```

Comment:
You come off as accusatory. Instead, what you want is to convery info to the user in a neutral way. Figure a way of doing this